### DISCUSSION TITLE
Model names and locations

### DISCUSSION THREAD

**Discussion by opatrascoiu** [dmn-tck/tck#686] [General]
I spent some time to investigate how we link the DMN and TCK files. Here are my findings.

The test files contain a tag called `<modelName>`. However, the actual value is a file name, the name of the file that contains the model (e.g. 0106-feel-ternary-logic.dmn). All the test files are consistent, the content of `<modelName>` is always a file name. 

The content of the tag `<name>` in the DMN files does not match the name of the file. The majority of the DMN files have `<name>` the same as the file name without extension. About 57 DMN files have names unrelated to the file name. For example, 0106-feel-ternary-logic.dmn, has the name 'feel-logic'.  

I can think of the following approach to deal with these inconsistencies and have a precise way of linking TCK and DMN files:
1. Keep the content of `<modelName>` in the TCK files as it is, the suffix of the URI for the location of the DMN file.
2. Change the inconsistent DMN files to have `<name>` the same as the file name without extension (logical link), or add a property locationURI in TDefintions (physical link) in all of them.

Open to other suggestions.

**Comment by baldimir** [comment]
PR opened for this: https://github.com/dmn-tck/tck/pull/690
